src/methods.jl

```diff
@@ -77,3 +74,54 @@ julia> subtract_overscan(frame, (:, 4:5), dims = 2)
 * [`subtract_overscan!`](@ref)
 """
 subtract_overscan(frame::AbstractArray, idxs; dims = axes_min_length(idxs)) = subtract_overscan!(deepcopy(frame), idxs, dims = dims)
+
+
+"""
+    flat_correct!(frame::AbstractArray, flat_frame::AbstractArray; norm_value = mean(flat_frame))
+
+In-place version of [`flat_correct`](@ref)
+"""
+function flat_correct!(frame::AbstractArray, flat_frame::AbstractArray; norm_value = mean(flat_frame))
+    size(frame) != size(flat_frame) && error("size of frame and flat frame are not same")
+    norm_value < 0 && error("norm_value must be greater than 0")
```

Comment:
I think we can't accept zero, right? The error message looks correct (but I'd say "positive"), but the check isn't consistent with it

Reply:
Yes, we should do `norm_value > 0 || error`

Reply:
Just added an equal to sign as well!